hasura/migrations/default/1648654670645_new_amount_and_distribution_id/down.sql

```diff
@@ -0,0 +1,15 @@
+
+-- Could not auto-generate a down migration.
```

Comment:
Wondering if this would cause issues if we wanted to rollback. Not an SQL pro so not sure, but if we can would be good to write manual down migrations here

Reply:
Yeah, I saw this and at the time I felt that it wasn't that big a deal cause worst case, you just have to write alter table statements, remove constraints and drop tables/columns. not a huge lift but open to hearing if this should be included now.